### PR TITLE
new icon: emacs (original)

### PR DESCRIPTION
**Pull request by theblobscp**
You can't have vim and not include emacs

### CODE REVIEW

**Comment by theblobscp** [comment]
@Panquesito7 can you review this if you get a chance? Thanks

File: devicon.json [hunk 1]
```diff
@@ -1195,6 +1195,16 @@
         "color": "#34495E",
         "aliases": []
     },
+    {
+        "name": "emacs",
+        "tags": ["editor"],
+        "versions": {
+            "svg": ["original"],
+            "font": ["original"],
```
Comment (Panquesito7) [inline devicon.json:1203]:
```suggestion
            "svg": [
                "original"
            ],
            "font": [
                "original"
            ],
```

File: icons/emacs/emacs-original.svg [hunk 2]
```diff
@@ -0,0 +1 @@
...7344-0.19141 0.027344-0.47266 0-0.625-0.03125-0.15625-0.054688 0-0.054688 0.34375m-25.062 2.2773c0.22266 0.027344 0.5625 0.027344 0.75 0s0.003906-0.050782-0.41016-0.050782-0.56641 0.023438-0.33984 0.050782m-2.5039 0.125c0.15625 0.027344 0.40625 0.027344 0.5625 0 0.15625-0.03125 0.027344-0.054688-0.28125-0.054688s-0.4375 0.023438-0.28125 0.054688m-3.375 0.25c0.15625 0.027344 0.40625 0.027344 0.5625 0 0.15625-0.03125 0.027344-0.054688-0.28125-0.054688s-0.4375 0.023438-0.28125 0.054688m-9 0.125c0.15625 0.027344 0.40625 0.027344 0.5625 0 0.15625-0.03125 0.027344-0.054688-0.28125-0.054688s-0.4375 0.023438-0.28125 0.054688m6.6875 0c0.1875 0.027344 0.5 0.027344 0.6875 0s0.035156-0.050782-0.34375-0.050782-0.53125 0.023438-0.34375 0.050782m-20.031 9.9141c-0.13281 0.082032-0.03125 0.11719 0.36328 0.11719 0.59375 0.003906 0.63281-0.042969 0.13672-0.16016-0.17188-0.042969-0.39844-0.023438-0.5 0.042968m9.4062-0.042968c0.12109 0.03125 0.31641 0.03125 0.4375 0s0.023438-0.058594-0.21875-0.058594-0.33984 0.027344-0.21875 0.058594m-2.375 0.12891c0.19141 0.027344 0.47266 0.027344 0.625-0.003906 0.15625-0.027344 0-0.050782-0.34375-0.050782s-0.46875 0.027344-0.28125 0.054688" fill="#a595cf"/></g></svg>
```
Comment (Panquesito7) [inline icons/emacs/emacs-original.svg:1]:
```suggestion
<svg width="128" height="128" viewBox="0 0 128 128" version="1.0" xmlns="http://www.w3.org/2000/svg"><g fill-rule="evenodd"><path d="M59.062 7.133C18.179 10.887-5.485 55.195 14.003 91.488c17.172 31.977 59.629 39.766 86.746 15.922 30-26.383 24.613-74.762-10.438-93.758-9.223-4.996-21.02-7.457-31.25-6.52m21.246 13.801c9.605 1.836 13 13.012 5.129 16.883-2.074 1.02-3.18 1.184-9.75 1.434-6.688.258-12.887 1.094-18.051 2.437-5.027 1.309-6.57 3.696-3.977 6.149 1.453 1.375 2.922 2.16 12.652 6.765a1938.71 1938.71 0 0 1 15.555 7.434c1.547.75 1.73.61-1.93 1.465-20.57 4.793-28.124 7.738-32.374 12.613-4.059 4.66-2.102 7.848 6.109 9.945 5.543 1.418 8.914 1.684 26.203 2.07 11 .246 15.883 1.262 17.059 3.551 2.559 4.985-15.867 11.102-40.902 13.582-2.898.285-12.578.48-12.094.242.137-.07 2.5-.636 5.25-1.261 19.09-4.34 30.512-7.633 30.109-8.676-.2-.52-5.242-.688-9.734-.332-12.945 1.031-20 .031-31.375-4.45-15.047-5.929-17.184-15.476-5.153-23.03 7.985-5.012 17.832-7.54 28.684-7.36l2.535.043-1.285-.414C52.6 56.68 43.002 51.16 38.541 45.98c-3.011-3.504-3.62-5.246-2.574-7.375 1.555-3.16 5.207-4.312 15.594-4.914 6.191-.36 17.562-1.59 20.938-2.266 4.324-.867 5.793-2.8 3.102-4.093-1.653-.793-3.887-.973-13.285-1.067l-7.746-.078 1.683-.48c16.11-4.582 20.496-5.453 24.055-4.774" fill="#8562bb"/><path d="M75.75 20.836c-1.73.281-5.16 1.074-9.16 2.113-3.317.864-11.621 3.164-11.695 3.239-.031.03 3.066.058 6.886.058 10.902.004 13.414.293 14.707 1.692 1.434 1.558-.343 2.93-4.757 3.68-4.063.69-14.168 1.784-19.105 2.07-11.727.671-15.223 1.77-16.77 5.265-2.293 5.184 10.434 15.391 25.578 20.508l3.129 1.059-1.883-.067c-8.125-.285-16.703 1.235-23.555 4.168-18.09 7.739-18.371 19.34-.64 26.234 10.538 4.098 18.354 5.336 28.515 4.52 8.242-.664 12.902-.488 12.305.46-.738 1.165-12.855 4.544-31.598 8.81-3.988.906-4.008.949-.457.968 14.102.07 36.031-4.062 45.062-8.492 7.824-3.836 6.086-7.387-4.187-8.55-2.38-.27-6.29-.423-12.875-.509-15.86-.203-23.102-1.406-28.043-4.664-4.86-3.199.925-10.387 11.383-14.133 4.082-1.46 15.164-4.457 23.199-6.273.675-.152 1.18-.317 1.125-.367-.118-.102-5.047-2.485-10.29-4.973-16.163-7.672-16.886-8.035-18.53-9.336-3.36-2.66-1.903-5.289 3.73-6.738 5.563-1.43 11.602-2.2 19.344-2.457 7.95-.266 11.184-1.836 12.27-5.946 1.805-6.84-5.734-13.64-13.688-12.34" fill="#fbfbfb"/><path d="M52.312.148c-9.59 2.344-16.125 4.938-22.562 8.95-8.863 5.53-16.375 13.348-21.523 22.402C4.825 37.484 2.106 44.773.383 52.535L0 54.26v19.73l.371 1.723c5.747 26.715 22.633 44.605 48.88 51.793l1.816.496L64 127.972l12.938-.035 1.937-.586c9.207-2.793 14.504-5.188 20.812-9.39 11.836-7.892 21.305-20.77 25.293-34.399 1.47-5.023 2.246-8.457 2.707-11.94.266-2 .27-12.806.008-14.876-3.09-24.414-19.098-45-41.445-53.3C83.216 2.315 78.353.866 75.337.19 74.325-.035 53.228-.074 52.314.148m15.13 3.61c12.59.824 25.972 6.172 34.288 13.699l.668.605-1.199.977c-1.406 1.144-1.367 1.004-.492 1.71 1.078.872 4.344 4.157 5.715 5.754 17.402 20.234 18.648 49.98 2.992 71.496-16.125 22.156-45.363 29.625-69.98 17.883-10.2-4.867-19.176-13.227-24.81-23.102l-.784-1.378-1.192.984c-.656.543-1.238.988-1.293.988-.676 0-4.504-9.61-5.672-14.25C-4.338 39.401 27.045 1.101 67.443 3.757" fill="#461e69"/><path d="M60.625 3.707C22.863 5.727-3.652 42.125 5.684 79.125c1.168 4.64 4.996 14.25 5.672 14.25.055 0 .637-.441 1.289-.984 1.04-.86 1.207-.957 1.332-.782.078.114-.27-.64-.777-1.672C.884 64.894 7.754 35.21 29.75 18.414c20.34-15.527 49.083-15.195 68.97.797.703.567 1.28.977 1.28.918 0-.062.54-.554 1.2-1.09l1.2-.976-.669-.606c-10-9.046-26.406-14.535-41.105-13.75m39.5 16.582c0 .024.184.204.407.407l.406.367-.367-.406c-.344-.38-.446-.465-.446-.368m1.438 1.336c.34.344.644.625.68.625.03 0-.215-.28-.555-.625s-.645-.625-.68-.625c-.031 0 .215.282.555.625m1.812 1.688c.442.445.832.812.867.812.036 0-.3-.367-.742-.812s-.832-.813-.867-.813c-.035 0 .3.368.742.813m1.871 1.969c.305.328.575.593.594.593.094 0-.008-.113-.558-.628l-.594-.56.558.595m.754.883c0 .023.184.203.407.406l.406.367-.367-.406c-.344-.38-.446-.465-.446-.367m14.027 32.21c0 .243.028.34.059.22s.031-.317 0-.438-.059-.024-.059.219m.13 1.437c0 .273.027.371.058.219a1.421 1.421 0 0 0-.004-.5c-.031-.121-.055.008-.055.281m.137 2.5c0 .617.023.887.047.598a8.29 8.29 0 0 0 .004-1.125c-.028-.328-.051-.09-.051.527m0 3.75c0 .617.023.887.047.598a8.29 8.29 0 0 0 .004-1.125c-.028-.328-.051-.09-.051.527m-.137 2.5c0 .273.028.371.059.219a1.421 1.421 0 0 0-.004-.5c-.031-.121-.055.008-.055.281m-97.844 34.688c.34.344.645.625.68.625.031 0-.215-.281-.555-.625s-.644-.625-.68-.625c-.03 0 .215.281.555.625m81.184 1.594-1.433 1.469 1.468-1.434c.809-.793 1.47-1.453 1.47-1.469 0-.09-.169.07-1.505 1.434m-78.184 1.406c.34.344.645.625.68.625.031 0-.215-.281-.555-.625s-.644-.625-.68-.625c-.03 0 .215.281.555.625m33.47 14.961c.12.031.316.031.437 0s.023-.059-.219-.059-.34.028-.219.059m1.875.129c.188.027.5.027.688 0s.035-.05-.344-.05-.531.023-.344.05m5.442 0c.222.027.562.027.75 0s.004-.05-.41-.05-.567.023-.34.05" fill="#5b2384"/><path d="M77.09 20.715c.223.027.617.027.875 0 .262-.027.078-.047-.402-.05-.48 0-.696.023-.473.05m-8.934 5.625c.258.027.68.027.938 0s.047-.047-.469-.047-.726.02-.469.047m2.75.121c.121.031.317.031.438 0s.023-.059-.219-.059-.34.028-.219.06M89.66 30.75c0 .309.024.438.055.281a1.939 1.939 0 0 0 0-.562c-.031-.156-.055-.027-.055.281m-.004 1.063c0 .273.028.37.06.218a1.421 1.421 0 0 0-.005-.5c-.031-.12-.055.008-.055.282m-32.5 1.523c.122.031.317.031.438 0s.023-.059-.219-.059-.34.028-.219.06m-1.25.124c.122.031.317.031.438 0s.023-.059-.219-.059-.34.028-.219.06m-2 .128c.188.027.5.027.688 0s.035-.05-.344-.05-.531.023-.344.05m-2.125.125c.192.027.473.027.625 0 .157-.031 0-.055-.343-.055s-.469.028-.282.055m-2.062.125c.156.027.406.027.562 0 .157-.031.028-.055-.28-.055s-.438.024-.282.055m-1.75.125c.156.027.383.027.5-.004.121-.031-.008-.055-.281-.055s-.371.028-.219.059m-1.438.121c.122.031.317.031.438 0s.023-.059-.219-.059-.34.028-.219.06m32.816 5.003c.223.027.56.027.75 0 .188-.027.004-.05-.41-.05s-.566.023-.34.05m-4.128.125c.222.027.59.027.812 0s.04-.05-.406-.05-.63.023-.406.05m-2.25.125c.156.027.382.027.5-.004.12-.031-.008-.055-.282-.055s-.37.028-.218.059m-1.688.121c.121.031.317.031.438 0s.023-.059-.22-.059-.339.028-.218.06m-18.164 4.382-.43.469.47-.43c.257-.238.468-.45.468-.469 0-.097-.106-.008-.508.43m-.84 1.656c0 .242.028.34.059.219s.031-.317 0-.438-.059-.023-.059.22m-12.59 1.75c.477.48.899.874.93.874.035 0-.328-.394-.805-.875s-.894-.875-.93-.875c-.035 0 .329.395.805.875m13.75.5c.27.274.516.5.551.5.035 0-.156-.226-.426-.5s-.515-.5-.55-.5c-.036 0 .156.227.425.5m9.438 12.625-1.063.078 1 .024c.551.011 1.086-.036 1.188-.102s.133-.113.062-.101c-.07.011-.601.058-1.187.101m-4.531.09c.293.027.77.027 1.062 0 .293-.024.055-.047-.531-.047s-.824.023-.531.047m-2.375.125c.156.027.406.027.562 0 .156-.031.027-.055-.281-.055s-.438.024-.281.055m-1.438.121c.121.031.316.031.438 0s.023-.059-.22-.059-.339.028-.218.059M25.293 78.25c0 .516.02.727.047.469s.027-.68 0-.938-.047-.047-.047.469m20.11 2.25c0 .242.026.34.058.219s.03-.317 0-.438-.059-.023-.059.219m18.816 7.215c.156.027.383.027.5-.004.121-.031-.008-.055-.281-.055s-.371.028-.219.059m1.875.125c.156.027.406.027.563 0 .156-.031.027-.055-.282-.055s-.437.024-.28.055m2.812.125c.328.027.859.027 1.187 0 .328-.024.059-.043-.594-.043s-.922.02-.593.043m7.132.129c.536.023 1.38.023 1.871 0 .497-.024.059-.043-.972-.043s-1.438.02-.899.043m6.555.12c.223.028.59.028.813 0s.039-.05-.407-.05-.629.023-.406.05m2.75.126c.156.027.383.027.5-.004.121-.031-.008-.055-.281-.055s-.371.028-.219.059m11.816 4.473c0 .343.027.468.055.28a2.386 2.386 0 0 0 0-.624c-.032-.156-.055 0-.055.344M72.097 95.09c.223.027.563.027.75 0s.004-.051-.41-.051-.567.023-.34.05m-2.504.126c.156.027.406.027.563 0 .156-.031.027-.055-.282-.055s-.437.024-.28.055m-3.376.25c.156.027.406.027.563 0 .156-.031.027-.055-.282-.055s-.437.024-.28.055m-9 .125c.155.027.405.027.562 0 .156-.031.027-.055-.282-.055s-.437.024-.28.055m6.687 0c.187.027.5.027.687 0s.035-.051-.344-.051-.53.023-.343.05m-20.031 9.915c-.133.082-.032.117.363.117.594.004.633-.043.137-.16-.172-.043-.399-.023-.5.043m9.406-.043c.12.031.316.031.437 0s.024-.059-.218-.059-.34.028-.22.059m-2.374.129c.191.027.473.027.625-.004.156-.027 0-.05-.344-.05s-.469.027-.281.054" fill="#a595cf"/></g></svg>

```

**Review by Panquesito7 — CHANGES_REQUESTED** [review]

**Comment by Thomas-Boi** [comment]
Hey @theblobscp,

The icon overall looks good. We just need a few more changes before we can accept it:
- The Emacs's border doesn't look smooth. Can you smooth it out some how? This should be achievable using `<circle>` elements or just a circular path.
- We will also need a `font` friendly version for the icon, so a `plain` or `line` version is necessary. You can start with having just a circle and make the `E` transparent.

**Comment by theblobscp** [comment]
Sorry, can't do that. I kinda sick with svg

**Comment by Thomas-Boi** [comment]
In that case, how did you make this SVG? Perhaps can you look around for an already premade version? If you can do that, I can help with making it a `plain` version (although I think it could be a learning experience if you have Inkscape)

**Comment by theblobscp** [comment]
Usually I just convert a png to svg, and find online tools to fix everything else

**Comment by Thomas-Boi** [comment]
Since we aren't sure where to move on from here, I'll close this PR for now. The SVG unfortunately is not very high quality (the border is fuzzy). This won't make for a good icon.

Perhaps you can open an Issue for emacs (if one hasn't exist yet)? That might alert someone and they can work/find a suitable SVG.

**Pull request closed without merging**